egs2/TEMPLATE/asr1/pyscripts/feats/ssl_feature_utils.py

```diff
@@ -51,7 +51,9 @@ def dump_feature(
             if is_scipy_wav_style(mat):
                 # If data is sound file, then got as Tuple[int, ndarray]
                 rate, mat = mat
-                mat = mat.astype(np.float64, order="C") / 32768.0
+                mat = mat.astype(np.float64, order="C")
```

Comment:
can you also update the similar parts in `dump_km_label.py`?

Reply:
sounds good

Reply:
I update this file in #5285. The espnet data reader is used. Probably this will not be a problem anymore.

Reply:
Sounds cool! I can remove this part then

Reply:
It would be good if you can double check it.